src/class-parsely.php

```diff
@@ -1069,6 +1073,22 @@ function( $v ) {
 		}
 	}
 
+	public static function get_asset_cache_buster() {
+		static $cache_buster;
+		if ( isset( $cache_buster ) ) {
+			return $cache_buster;
+		}
+
+		$cache_buster = defined( 'WP_DEBUG' ) && WP_DEBUG && empty( 'WP_TESTS_DOMAIN' ) ? wp_rand() : PARSELY_VERSION;
+
+		/**
+		 * Filters the cache buster value for linked scripts and styles.
+		 *
+		 * @param string $cache_buster Plugin version, unless WP_DEBUG is defined and truthy, and tests are not running.
```

Comment:
```suggestion
		 * Filters the cache buster value for linked scripts and styles.
		 *
		 * @since 2.5.0
		 *
		 * @param string $cache_buster Plugin version, unless WP_DEBUG is defined and truthy, and tests are not running.
```